File: 1.两数之和.cpp
/*
 * @Author:  ber bai5775@outlook.com
 * @Date: 2024-07-16 14:46:41
 * @LastEditors:  ber bai5775@outlook.com
 * @LastEditTime: 2024-07-23 10:27:57
 * @FilePath: 1.两数之和.cpp
 * @Description: 
 * 
 * Copyright (c) 2024 by bai5775@outlook.com, All Rights Reserved. 
 */
/*
 * @lc app=leetcode.cn id=1 lang=cpp
 *
 * [1] 两数之和
 */

// @lc code=start
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i, j, n=nums.size();
        for(i=0;i<n;i++) {
            for(j=i+1;j<n;j++) {
                if(target == nums[i]+nums[j]) {
                    return {i,j};
                }
            }
        }
        return {};
    }
};
// @lc code=end

File: 1.两数之和.py
'''
Author:  ber bai5775@outlook.com
Date: 2024-07-18 08:39:21
LastEditors:  ber bai5775@outlook.com
LastEditTime: 2024-07-23 10:28:11
FilePath: 1.两数之和.py
Description: 

Copyright (c) 2024 by bai5775@outlook.com, All Rights Reserved. 
'''
#
# @lc app=leetcode.cn id=1 lang=python3
#
# [1] 两数之和
#

# @lc code=start
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n = len(nums)
        for i in range(n):
            for j in range(i+1, n):
                if nums[i]+nums[j]==target:
                    return [i,j]
        return []
# @lc code=end

